portfolio/src/main/webapp/aboutMe.html

```diff
@@ -0,0 +1,84 @@
+<!--This is the About me Page. 
+    Created by Adedeji Adebisi 
+    Date: 05/25/2020
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Adedeji Portfolio</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!--Reference to bootstrap css stylesheet -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="aboutMe.css">
```

Comment:
If style.css has a lot of style that is in all 3 css files, you can include both CSS files. The last ones always take priority over the first ones, so you can override styles if desired too. This way you have one file with common things and not have to make a change in many areas.